tools/c7n_azure/c7n_azure/resources/cost_management_export.py

```diff
@@ -0,0 +1,163 @@
+# Copyright 2019 Microsoft Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from c7n_azure.actions.base import AzureBaseAction
+from c7n_azure.provider import resources
+from c7n_azure.query import QueryResourceManager
+from c7n_azure.utils import ThreadHelper
+
+from c7n.filters import Filter
+from c7n.utils import get_annotation_prefix as gap

```

Comment:
`gap` is a weird alias, because semantically it has a meaning in english sounds like a `keyword`

Reply:
Yeah, I picked it because it almost fits to the usages & `get_annotation_prefix` is really long. Do you think it is better to just use `get_annotation_prefix` (or some other alias?)

Reply:
I would favor `get_annotation_prefix` over `gap` for readability.